### PR TITLE
Remove minimum db version info in 4.8 release notes

### PR DESCRIPTION
**Pull request by EFM-Bobby**
We don't put supported database version information into the release notes -- users should check the platform compatibility page instead.

Adding it here was my mistake last time around, and the information is wrong now that v11 has gone out of support.

### CODE REVIEW

**Review by anilkumar29 — APPROVED** [review]

**Comment by EFM-Bobby** [comment]
@nidhibhammar Hi, can you ok this? Anil is the PM for efm and ok'ed the change, but I need a code owner to review before I can merge (and github suggested you).

**Review by djw-m — APPROVED** [review]
LGTM

**Pull request merged**
